fastly/stats_historical_test.go

```diff
@@ -85,6 +85,31 @@ func TestClient_GetStatsJSON(t *testing.T) {
 	}
 }
 
+func TestClient_GetAggregateJSON(t *testing.T) {
+	t.Parallel()
+
+	var ret struct {
+		Status string `json:"status"`
+	}
+
+	var err error
+	record(t, "stats/aggregate", func(c *Client) {
+		err = c.GetAggregateJSON(&GetAggregateInput{
+			From:   ToPointer("15 minutes ago"),
+			To:     ToPointer("now"),
+			By:     ToPointer("minute"),
+			Region: ToPointer("usa"),
+		}, &ret)
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if ret.Status != "success" {
+		t.Fatalf("got RenameStatus=%q, want %q", ret.Status, "success")
```

Comment:
I presume this is a typo and should be...

```suggestion
		t.Fatalf("got status=%q, want %q", ret.Status, "success")
```